metricbeat/docs/modules/elasticsearch.asciidoc

```diff
@@ -38,6 +38,9 @@ metricbeat.modules:
     #- ml_job
   period: 10s
   hosts: ["localhost:9200"]
+  #username: "elastic"
+  #password: "changeme"
+  #ssl.certificate_authorities: [ "/etc/pki/root/ca.pem" ]
```

Comment:
Some lines below there are links to common HTTP and TLS settings documentation, do you think it is still needed to add this example here? I wonder if then we should cover several scenarios.